client/lib/feed-stream-store/feed-stream.js

```diff
@@ -383,14 +381,20 @@ export default class FeedStream {
 					// we don't have the information to render this x-post
 					return false;
 				}
-				const isFollowing = FeedSubscriptionStore.getIsFollowingBySiteUrl( xPostMetadata.siteURL );
+				// TODO: @blowery / @bluefuton -- how can we solve this? Does this solution address the issue
```

Comment:
I didn't realize this file hadn't been prettified before.  384-->390 are the only lines i actually changed